vector-app/build.gradle

```diff
@@ -31,20 +31,12 @@ knit {
 }
 
 // Note: 2 digits max for each value
-<<<<<<< HEAD
 ext.versionMajor = 2
-=======
-ext.versionMajor = 1
->>>>>>> v1.5.2
 ext.versionMinor = 5
 // Note: even values are reserved for regular release, odd values for hotfix release.
 // When creating a hotfix, you should decrease the value, since the current value
 // is the value for the next regular release.
-<<<<<<< HEAD
 ext.versionPatch = 0
```

Comment:
you forgot to increment this digit after the 2.5.0, you should consider it after the rebase.